PHP 7.2 compatibility

When I try to compile php_v8js.dll on Windows for PHP 7.2 (alpha 1), I run into the following errors:
```
N:\php-sdk\php72dev
$ nmake -i php_v8js.dll | grep error

Microsoft (R) Program Maintenance Utility Version 14.10.25019.0
Copyright (C) Microsoft Corporation.  All rights reserved.

ext\v8js\v8js_class.cc(540): error C2065: 'ZEND_ACC_CLONE': undeclared identifier
ext\v8js\v8js_object_export.cc(339): error C2065: 'ZEND_ACC_CLONE': undeclared identifier
ext\v8js\v8js_object_export.cc(542): error C2065: 'ZEND_ACC_CLONE': undeclared identifier
ext\v8js\v8js_class.cc(540): error C2065: 'ZEND_ACC_CLONE': undeclared identifier
ext\v8js\v8js_object_export.cc(339): error C2065: 'ZEND_ACC_CLONE': undeclared identifier
ext\v8js\v8js_object_export.cc(542): error C2065: 'ZEND_ACC_CLONE': undeclared identifier
LINK : fatal error LNK1181: cannot open input file 'N:\php-sdk\php72dev\x64\Release_TS\ext\v8js\v8js_class.obj'
mt.exe : general error c101008d: Failed to write the updated manifest to the resource of file "N:\php-sdk\php72dev\x64\Release_TS\php_v8js.dll". The system cannot find the file specified.
```
The V8 libs were compiled with VC15 as well.

--- a/v8js_class.cc
+++ b/v8js_class.cc
@@ -537,7 +537,7 @@ static PHP_METHOD(V8Js, __construct)
 			continue;
 		}
 
-		if ((method_ptr->common.fn_flags & (ZEND_ACC_CTOR|ZEND_ACC_DTOR|ZEND_ACC_CLONE)) != 0) {
+		if ((method_ptr->common.fn_flags & (ZEND_ACC_CTOR|ZEND_ACC_DTOR)) != 0) {
 			/* no __construct, __destruct(), or __clone() functions */
 			continue;
 		}

--- a/v8js_object_export.cc
+++ b/v8js_object_export.cc
@@ -336,7 +336,7 @@ static void v8js_named_property_enumerator(const v8::PropertyCallbackInfo<v8::Ar
 			/* Allow only public methods */
 			continue;
 		}
-		if ((method_ptr->common.fn_flags & (ZEND_ACC_CTOR|ZEND_ACC_DTOR|ZEND_ACC_CLONE)) != 0) {
+		if ((method_ptr->common.fn_flags & (ZEND_ACC_CTOR|ZEND_ACC_DTOR)) != 0) {
 			/* no __construct, __destruct(), or __clone() functions */
 			continue;
 		}
@@ -539,7 +539,7 @@ static void v8js_fake_call_impl(const v8::FunctionCallbackInfo<v8::Value>& info)
 
 	if (method_ptr == NULL ||
 		(method_ptr->common.fn_flags & ZEND_ACC_PUBLIC) == 0 ||
-		(method_ptr->common.fn_flags & (ZEND_ACC_CTOR|ZEND_ACC_DTOR|ZEND_ACC_CLONE)) != 0) {
+		(method_ptr->common.fn_flags & (ZEND_ACC_CTOR|ZEND_ACC_DTOR)) != 0) {
 		error_len = spprintf(&error, 0,
 			"%s::__call to %s method %s", ce->name,
 			(method_ptr == NULL) ? "undefined" : "non-public", method_name);
@@ -619,7 +619,7 @@ v8::Local<v8::Value> v8js_named_property_callback(v8::Local<v8::String> property
 		 (method_ptr = reinterpret_cast<zend_function *>
 		  (zend_hash_find_ptr(&ce->function_table, method_name))) &&
 		 ((method_ptr->common.fn_flags & ZEND_ACC_PUBLIC) != 0) && /* Allow only public methods */
-		 ((method_ptr->common.fn_flags & (ZEND_ACC_CTOR|ZEND_ACC_DTOR|ZEND_ACC_CLONE)) == 0) /* no __construct, __destruct(), or __clone() functions */
+		 ((method_ptr->common.fn_flags & (ZEND_ACC_CTOR|ZEND_ACC_DTOR)) == 0) /* no __construct, __destruct(), or __clone() functions */
 		 ) || (method_ptr=NULL, is_magic_call)
 	) {
 		if (callback_type == V8JS_PROP_GETTER) {

